src/backend/src/prisma/schema.prisma

```diff
@@ -701,6 +701,8 @@ model Team_Type {
   iconName       String
   designReviews  Design_Review[]
   team           Team[]
+  description    String
+  image          String?
```

Comment:
Rename to `imageFileId` or something so it doesn't look like we're saving a string representation of an image